fido-key-manager/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "fido-key-manager"
+version = "0.1.0"
+authors = ["Michael Farrell <micolous+git@gmail.com>"]
+categories = ["authentication"]
+description = "Management CLI for FIDO/CTAP 2.x authenticators"
+edition = "2021"
+keywords = ["ctap", "fido", "webauthn"]
```

Comment:
We should probably start sticking passkeys in the keywords just to get that sweet sweet SEO on crates.io.

Reply:
Done